src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Client/ICommitmentsApiClient.cs

```diff
@@ -35,5 +35,6 @@ public interface ICommitmentsApiClient
         Task<AccountResponse> GetAccount(long accountId, CancellationToken cancellationToken = default);
         Task<GetApprenticeshipResponse> GetApprenticeship(long apprenticeshipId, CancellationToken cancellationToken = default);
         Task<GetPriceEpisodesResponse> GetPriceEpisodes(long apprenticeshipId, CancellationToken cancellationToken = default);
+        Task<GetApprenticeshipUpdatesResponse> GetApprenticeshipUpdates(GetApprenticeshipUpdateRequest request, CancellationToken cancellationToken = default);
```

Comment:
Minor, but the request name should be pluralised as it's a request for many things (potentially).